cmd/operator/main.go

```diff
@@ -78,11 +82,17 @@ func Main() int {
 
 	// Print params configured
 	logrus.Info("Using Variables:")
+	logrus.Infof("   kubecfg-file: %s", kubeCfgFile)
```

Comment:
This could be a security issue if you print out the tokens and they get brought into a logging system.

Reply:
Good point, fixed.